### PR TITLE
Major Changeling Tweaks

### PR DESCRIPTION
**Pull request by TheDanseMacabre**
Minor antagonist tweaks to changeling (not actually minor)

### CODE REVIEW

**Comment by TheDanseMacabre** [comment]
!wip

**Comment by TheDanseMacabre** [comment]
!review

**Review by Alberyk — APPROVED** [review]

**Review by Arrow768 — APPROVED** [review]
I guess we’ll see how that goes

**Review by SleepyGemmy — CHANGES_REQUESTED** [review]
changelog required.

**Comment by Arrow768** [comment]
> changelog required.

Good catch

File: html/changelogs/dansemacabre-horrorform.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+# Optional: Remove this file after generating master changelog.  Useful for PR changelogs that won't get used again.
+delete-after: True
+
+# Any changes you've made.  See valid prefix list above.
+# INDENT WITH TWO SPACES.  NOT TABS.  SPACES.
+# SCREW THIS UP AND IT WON'T WORK.
+# Also, all entries are changed into a single [] after a master changelog generation. Just remove the brackets when you add new entries.
+# Please surround your changes in  double quotes ("), as certain characters otherwise screws up compiling. The quotes will not show up in the changelog.
+changes:
+  - balance: "Changeling horror form has been improved."
```
Comment (SleepyGemmy) [inline html/changelogs/dansemacabre-horrorform.yml:41]:
```suggestion
  - balance: "Changeling horror form has been tweaked. It is now irreversible. Chem cost increased from 40 to 50. Health increased from 750 to 1250. Damage increased from 30 to 45. 30 AP added to attacks. Minimum oxygen changed from 0 to 5."
```

Comment (SleepyGemmy) [inline html/changelogs/dansemacabre-horrorform.yml:41]:
```suggestion
  - balance: "Changeling horror form has been tweaked. It is now irreversible. Genome cost decreased from 15 to 10. Chem cost increased from 40 to 50. Health increased from 750 to 1250. Damage increased from 30 to 45. 30 AP added to its melee attacks. Minimum oxygen required changed from 0 to 5."
```

**Review by SleepyGemmy — CHANGES_REQUESTED** [review]

**Review by SleepyGemmy — APPROVED** [review]

**Pull request merged**